django_project/changes/feeds/entry.py

```diff
@@ -122,7 +122,11 @@ def item_description(self, item):
         :returns: description of the Entry
         :rtype: str
         """
-        return '<p>'+item.description+'</p><p><img src="'+settings.MEDIA_URL+item.image_file.name+'"/></p>'
+
+        return '<p>' + item.description + \
```

Comment:
I know this is inherited code (i.e. you did not write it like this) but as a general pattern it is nicer to first assign the result to a variable and then return the var. The reason I prefer that is if you are debugging it is easier to evaluate what is being returned when stepping through the code.